de/manual/ch_GUIMenus.xml

```diff
@@ -6658,535 +6658,1179 @@ url="&url-docs-de;guide/chapter_txns.html#txns-regstyle1"> &app; Kurs und Konzep
+                  </para>
+                </entry>
+
+                <entry namest="c2" nameend="c3">
+                  <para>Löscht die Buchung dieser Rechnung und ermöglicht dadurch eine erneute Bearbeitung.
+                  </para>
+                </entry>
+              </row>
+
+              <row>
+                <entry>
+                  <para><guibutton>Rechnung bezahlen</guibutton>
+                  </para>
+
+                  <para><guibutton>Auslagenerstattung bezahlen</guibutton>
+                  </para>
+                </entry>
+
+                <entry namest="c2" nameend="c3">
+                  <para>Öffnet den Dialog <guilabel>Zahlung verarbeiten</guilabel> mit dem die Rechnung gebucht werden
+                    kann.
+                  </para>
+                </entry>
+              </row>
+            </tbody>
+          </tgroup>
+        </table>
+      </sect3>
+    </sect2>
+
+    <sect2 id="gui-business-overview">
+      <title>Übersicht von Kunden, Lieferanten und Mitarbeiter</title>
+
+      <para>Mit der <emphasis>Kundenübersicht</emphasis>, <emphasis>Lieferantenübersicht</emphasis> und
+        <emphasis>Mitarbeiterübersicht</emphasis> behalten Sie den Überblick über wichtige Daten
+        Ihrer Geschäftspartner und Mitarbeiter.
+      </para>
+
+      <sect3 id="gui-business-overview-display">
+        <title>Anzeigebereich</title>
 
-              <entry>
-                <para>Den Dialog für das Erstellen einer Rechnung öffnen. Der markierte Lieferant wird in den Dialog als
-                  Rechnungsempfänger eingetragen.
-                </para>
-              </entry>
-            </row>
+        <figure pgwide="1">
+          <title>Die Kundenübersicht</title>
+          <screenshot>
+            <mediaobject>
+              <imageobject role="html">
+                <imagedata fileref="figures/GUI_Business_Overview.png" srccredit="Christian Wehling" width="&img-w;" />
+              </imageobject>
 
-            <row>
-              <entry>
-                <para>&gmi.rep.bus.ven-rep;
-                </para>
-              </entry>
+              <imageobject role="fo">
+                <imagedata fileref="figures/GUI_Business_Overview.png" srccredit="Christian Wehling" />
+              </imageobject>
 
-              <entry>
-                <para>Den Bericht über den ausgewählten Lieferanten anzeigen.
+              <caption>
+                <para>Die Übersicht der Kundendaten in den <guilabel>Geschäftsfunktionen</guilabel>.
                 </para>
-              </entry>
-            </row>
+              </caption>
+            </mediaobject>
+          </screenshot>
+        </figure>
 
-            <row>
-              <entry>
-                <para>&gmi.bus.ven.pro-pay;
-                </para>
-              </entry>
+        <para>Um ein neues Fenster mit einer Übersicht von <emphasis>Kunden</emphasis>,
```

Comment:
Normalerweise `neuem Rreiter`. Ist das abhängig von einer Einstellung? Ich habe` […] in neuem Fenster öffnen` nur bei Bericht und Kontobuch gesehen.